survey.go

```diff
@@ -22,8 +20,8 @@ type Question struct {
 // Prompt is the primary interface for the objects that can take user input
 // and return a string value.
 type Prompt interface {
-	Prompt(*readline.Instance) (interface{}, error)
-	Cleanup(*readline.Instance, interface{}) error
+	Prompt() (interface{}, error)
```

Comment:
I think this is probably my favorite part of this whole PR - Thankyou so much for removing readline! While having to support individual systems like this is a bit scary, we'll handle the obscure edge cases over time. Overall this is yet another huge improvement